system/jlib/jexcept.cpp

```diff
@@ -519,12 +518,17 @@ void raiseAssertException(const char *assertion, const char *file, unsigned line
     s.append(file);
     s.append(", line ");
     s.append(line);
-    ERRLOG("%s",s.str());       // make sure doesn't get lost!
-    queryLogMsgManager()->flushQueue(10*1000);
+
+    if (queryLogMsgManager())
+    {
+        printStackReport();
+        ERRLOG("%s",s.str());       // make sure doesn't get lost!
+        queryLogMsgManager()->flushQueue(10*1000);
 #ifdef _DEBUG
-    // cause a breakpoint in the debugger if we are debugging.
-    //userBreakpoint();
+        // cause a breakpoint in the debugger if we are debugging.
+        //userBreakpoint();
 #endif
+    }
```

Comment:
I don't think so - nothing will have come out in the stack report either.

A good solution would be for DBGLOG to fallback to fprintf if the logMsgManager was NULL, but that would require a condition on every DBGLOG.  That might in itself be a good change.